README.md

```diff
@@ -45,7 +45,8 @@ curl -s https://raw.githubusercontent.com/cristianoliveira/ergo/master/install.s
 For windows you can find the executable in each [release](https://github.com/cristianoliveira/ergo/releases).
 
 ***Disclaimer:***
-I only use unix based systems on a daily basis, so I am not able to test each build :(
+
+I use unix based systems on a daily basis, so I can't test every build on every OS :(
```

Comment:
Actually, this is more related to Windows-based systems hehe

Reply:
Hahaha I saw that. Didn't want to make it so obvious like "I'm not testing exe, good luck"

Reply:
Ahhhhh I got ya! Good idea :)